src/checks/checks.ts

```diff
@@ -75,9 +75,12 @@ export default class Checks {
 
         if (!this.ownCheck) {
             let ownCheckName = await extractOwnCheckNameFromWorkflow();
-            let gitHubActionsBotId = GitHubActionsBotId;
 
-            this.ownCheck = this.allChecks.find(check => check.name === ownCheckName && check.app.id === gitHubActionsBotId);
+            this.ownCheck = this.allChecks.find(check => check.name === ownCheckName && check.app.slug === 'github-actions');
```

Comment:
Good catch, but could we export the app slug as a constant from the checksConstants.ts file in place of the App ID?

Reply:
Good call, this should be updated to use a constant now.